test/js-scripts/package.json

```diff
@@ -28,6 +28,12 @@
   },
   "scripts": {
     "forge-test-getSqrtPriceAtTick": "npx ts-node src/getSqrtPriceAtTick.ts",
-    "forge-test-getTickAtSqrtPrice": "npx ts-node src/getTickAtSqrtPrice.ts"
+    "forge-test-getTickAtSqrtPrice": "npx ts-node src/getTickAtSqrtPrice.ts",
+    "forge-test-mulDiv": "npx ts-node src/mulDiv.ts",
+    "forge-test-mulDivRoundingUp": "npx ts-node src/mulDivRoundingUp.ts"
+  },
+  "dependencies": {
+    "@uniswap/v3-sdk": "^3.11.2",
+    "jsbi": "^4.3.0"
```

Comment:
did you manually install this? Don't think we need to

Reply:
oh ty!